etc/profile-a-l/email-common.profile

```diff
@@ -14,6 +14,7 @@ noblacklist ${HOME}/.signature
 # when storing mail outside the default ${HOME}/Mail path, 'noblacklist' the custom path in your email-common.local
 # and 'blacklist' it in your disable-common.local too so it is kept hidden from other applications
 noblacklist ${HOME}/Mail
+noblacklist /var/lib/clamav
 noblacklist /var/mail
```

Comment:
Protect /var/run/clamav too.
Suggestion:

```
noblacklist /var/run/clamav
```

Add the counterpart `blacklist /var/run/clamav` to the var section in `disable-common.inc` to complete the protection logic.

Reply:
> Add the counterpart blacklist /var/run/clamav to the var section in disable-common.inc to complete the protection logic.

Isn't there already at line 234?

Reply:
Done in line 19